src/ethsign/ethsign.go

```diff
@@ -41,16 +41,13 @@ func rawSignHash(data []byte) []byte {
   return crypto.Keccak256([]byte(msg))
 }
 
-func getWallets(c *cli.Context, defaultKeyStores cli.StringSlice) []accounts.Wallet {
+func getWallets(c *cli.Context, defaultKeyStore string) []accounts.Wallet {
```

Comment:
I think this param is actually unused? Perhaps we can remove it from `getWallets` and `getWalletData`?